android/src/main/java/com/mr/flutter/plugin/filepicker/FilePickerPlugin.java

```diff
@@ -23,6 +23,7 @@
 import io.flutter.plugin.common.EventChannel;
 import io.flutter.plugin.common.MethodCall;
 import io.flutter.plugin.common.MethodChannel;
+import io.flutter.plugin.common.PluginRegistry.Registrar;
```

Comment:
Why would you bring V1 back even more? It's deprecated for good.
Correct solution: https://github.com/miguelpruivo/flutter_file_picker/pull/1644

Reply:
This package are missing so its showing error now its clear